stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -89,8 +89,11 @@ extension DoubleWidth : Comparable {
 extension DoubleWidth : Hashable {
   @_inlineable // FIXME(sil-serialize-all)
   public var hashValue: Int {
-    // FIXME: Align this computation with that of synthesized Hashable types.
-    return _mixInt(high.hashValue) ^ low.hashValue
+    var result = 0
+    result = _combineHashValues(result, _storage.high.hashValue)
+    result = _combineHashValues(result, _storage.low.hashValue)
+    result = _mixInt(result)
+    return result
```

Comment:
This seems overdone to me. It looks like `_combineHashValues` by itself is all we'd need here.

Reply:
Sure is, but it’s [what’s done for synthesized conformances](https://github.com/xwu/swift/blob/30d5e344b1464966c1fb77770d046574a97b7705/lib/Sema/DerivedConformanceEquatableHashable.cpp#L1030). Given that Tony’s goal is eventually to allow synthesis for types with tuple members, I think we would do well to line things up here so that this extension could potentially be seamlessly deleted in the future.